src/sdc11073/pysoap/msgfactory.py

```diff
@@ -53,7 +59,7 @@ def serialize_message(self, message: CreatedMessage, pretty=False,
         :return: bytes
         """
         p_msg = message.p_msg
-        nsh = self._sdc_definitions.data_model.ns_helper
+        nsh = self.ns_hlp
```

Comment:
why create new variable? check also other places

Reply:
Improves readability in my opinion.

Reply:
with `self._sdc_definitions.data_model.ns_helper` it really did, but now I guess its not needed anymore. but lets stick to it